packages/next/webpack.config.js

```diff
@@ -135,19 +135,18 @@ module.exports = ({ dev, turbo, bundleType, experimental }) => {
     optimization: {
       moduleIds: 'named',
       minimize: true,
-      // splitChunks: {
-      //   chunks: 'all',
-      // },
       concatenateModules: true,
       minimizer: [
         new TerserPlugin({
```

Comment:
Why don't we use our own `TerserWebpackPlugin` here? https://github.com/vercel/next.js/tree/canary/packages/next/src/build/webpack/plugins/terser-webpack-plugin